docker/dev.Dockerfile.alpine

```diff
@@ -0,0 +1,29 @@
+FROM python:3.7-alpine AS builder
+
+RUN addgroup -S zulipgr \
+    && adduser -S zulip -G zulipgr \
+    && apk add --no-cache git ca-certificates gcc musl-dev \
```

Comment:
These indents seem clearer; if we don't merge files, we could tidy the other files too.